### PR TITLE
update version with lodash fixes to master

### PR DESCRIPTION
**Pull request by smagesh2106**
Fixes for https://github.ibm.com/IBMPrivateCloud/roadmap/issues/44850

### CODE REVIEW

**Review by rashmi43 — COMMENTED** [review]
Please update the imageTag for platformAuthService as well. Please cross reference the issue.

File: deploy/olm-catalog/ibm-iam-operator/3.10.0/ibm-iam-operator.v3.10.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -1725,7 +1725,7 @@ spec:
                 - name: ICP_OIDCCLIENT_WATCHER_IMAGE
                   value: quay.io/opencloudio/icp-oidcclient-watcher:3.4.2
                 - name: IAM_POLICY_CONTROLLER_IMAGE
-                  value: quay.io/opencloudio/iam-policy-controller:3.4.0
+                  value: quay.io/opencloudio/iam-policy-controller:3.4.1
```
Comment (rashmi43) [inline deploy/olm-catalog/ibm-iam-operator/3.10.0/ibm-iam-operator.v3.10.0.clusterserviceversion.yaml:1728]: You need to update PAP and not policy controller

**Review by rashmi43 — APPROVED** [review]
Added the issue and updated the imageTags

**Review by pnpavan25 — APPROVED** [review]
lgtm

**Pull request merged**